adev/shared-docs/pipeline/guides/extensions/docs-code/docs-code.ts

```diff
@@ -41,7 +42,6 @@ export const docsCodeExtension = {
     const code = singleFileCodeRule.exec(src);
     const selfClosingCode = singleFileSelfClosingCodeRule.exec(src);
     const match = selfClosingCode ?? code;
```

Comment:
These lines can now be just:

```suggestion
    const match = singleFileCodeRule.exec(src);
```

Reply:
Updated